### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by jsilvestre**
I did it this week-end, thought it'd be nice to push a little bit forward what we did during the last sprint.

* `deploy` has a parameter to choose the app's slug, (useful for @m4dz :))
* `deploy` shows the URL where the application is accessible from
* new command: `db:name` to get which database the VM is using
* `db:name` is called at the end of `vm:start`
* improvement to `db:switch` so the code is less complicated, and works in more cases
* linted!

### CODE REVIEW

**Comment by poupotte** [comment]
It's ok for me.

**Comment by frankrousseau** [comment]
The Feeds developer told me that he has issues while using the deploy command. Can you test a little bit more? His other problems were fixed.

**Comment by jsilvestre** [comment]
This morning, @cylwin used it without any problem (the current published version, not my PR) . I need more information about the issue encountered to have a look at it.

**Comment by frankrousseau** [comment]
Ok. Thank you for the information.

File: src/database.coffee [hunk 1]
```diff
@@ -103,3 +110,23 @@ module.exports = class DatabaseManager
...
+        command = """
+        vagrant ssh -c "sudo cat #{CONTROLLER_CONFIG_PATH}"
+        """
+        exec command, (err, stderr, stdout) ->
+            try
+                config = JSON.stringify stdout
+                databaseName = config?['data-system']?['DB_NAME']
+
+                # default is cozy
+                databaseName = 'cozy' unless databaseName?
```
Comment (frankrousseau) [inline src/database.coffee]: Here is a better way to write it:

```coffeescript
databaseName ?= 'cozy'
```

File: src/helpers.coffee [hunk 2]
```diff
@@ -56,7 +56,10 @@ module.exports.promptPassword = (name) -> (cb) ->
...
-        home = process.env.HOME or process.env.HOMEPATH or process.env.USERPROFILE
-        return path.join(home, "#{name}.pid")
+        home =
+            process.env.HOME or
+            process.env.HOMEPATH or
+            process.env.USERPROFILE
+        return path.join home, "#{name}.pid"
     else
-        return path.join('/tmp', "#{name}.pid")
+        return path.join '/tmp', "#{name}.pid"
```
Comment (frankrousseau) [inline src/helpers.coffee]: I'm not sure about the slash for windows user.

Comment (jsilvestre) [inline src/helpers.coffee]: the condition is `if this is windows.....else return path.join '/tmp'...`

Comment (frankrousseau) [inline src/helpers.coffee]: Thx :). I read too fast.

File: src/database.coffee (hunk 1 above)
Comment (jsilvestre) [inline src/database.coffee]: I fixed it, it's better indeed

**Pull request merged**